OneCommunityGlobal/HighestGoodNetworkApp
# Shantanu Unit Test Cases for Profile Nav Dot Usemanagament componenet

# Description
Shantanu Unit Test Cases for Profile Nav Dot Component

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Write on terminal "npm test profilenavdot.test.jsx"
5. Check if there are no errors

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/094c4276-39ca-45d6-842c-194f74952c62)


## Note:
Include the information the reviewers need to know.
